src/lib/exporter/frameRenderer.test.ts

```
@@ -1,361 +1,372 @@
+					}
+
+					const canvas = createMockCanvas();
+					createdCanvases.push(canvas);
+					return canvas;
+				}),
+			},
+		});
+	});
+
+	it("clamps webcam sync seeks to the media duration", async () => {
+		const renderer = createRenderer() as any;
```

Biome (2.4.11)

[error] Unexpected any. Specify a different type.

(lint/suspicious/noExplicitAny)